### PR TITLE
fix(view): update Choose what to sync design

### LINKED ISSUE #3183: "Choose what to sync" design refresh
from @ryanfeeley:

image

### PR DESCRIPTION
**Pull request by vladikoff**
Fixes #3183

### CODE REVIEW

**Comment by vladikoff** [comment]
Blocked by final design review in : https://github.com/mozilla/fxa/pull/65

**Comment by vladikoff** [comment]
@shane-tomlinson we are waiting for some string changes, but otherwise this is ready for review

r?

File: app/scripts/templates/choose_what_to_sync.mustache [hunk 1]
```diff
@@ -3,8 +3,8 @@
 
   <header>
     <h1 id="fxa-choose-what-to-sync-header">
-      {{#t}}Choose What to Sync{{/t}}
-      <span class="email">for {{ email }}</span>
```
Comment (shane-tomlinson) [inline app/scripts/templates/choose_what_to_sync.mustache]: Oooh, this was never translated in the first place, doh!

**Comment by shane-tomlinson** [comment]
Looks good in languages if the assumption is the words will never be "too long". On mobile, if a word grows to be long, the layout goes from a 2 column display to a 1 column display. "Preferences" and "Addons" are separated with 2 items between then. @ryanfeeley - is this a problem?

I doctored Swedish or Danish to add some extra characters to Bookmarks.


#### mobile

screen shot 2015-10-28 at 14 02 44

#### desktop

screen shot 2015-10-28 at 14 03 12

None of the "long" languages that I've tested actually have a problem on mobile or desktop. Here is German.

#### mobile

screen shot 2015-10-28 at 14 08 18

#### desktop

screen shot 2015-10-28 at 14 08 01

Code looks good to me, I hand this over to @ryanfeeley to give a UX r+.

**Comment by ryanfeeley** [comment]
@shane-tomlinson Well spotted! Is there a way to do this so that the order is maintained? Instead of two columns, do each one as a floating block?

**Comment by vladikoff** [comment]
>  Is there a way to do this so that the order is maintained?

Sadly no, I talked to Flexbox gecko devs and this was the solution we came up with.

**Comment by ryanfeeley** [comment]
If add-ons were above preferences in the right column, would this solve it?

**Comment by vladikoff** [comment]
> If add-ons were above preferences in the right column, would this solve it?

Sure, then in the 2 col views both would be on the right, second col. Do you want that?

**Comment by ryanfeeley** [comment]
Yes! Matches sync prefs right now too, no? 
screenshot 2015-10-28 11 34 23

**Comment by vladikoff** [comment]
> Yes! Matches sync prefs right now too, no? 

:+1:

**Comment by vladikoff** [comment]
I updated the design spec so we keep the order in mind in the future: https://github.com/mozilla/fxa/commit/39ec152ede28340f7e8b430478eb1d959f67a569

**Comment by ryanfeeley** [comment]
:+1:

**Pull request merged**